scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/FilterView.java

```diff
@@ -297,6 +521,18 @@ public String getValue() {
             return textBox.getText();
         }
 
+        public void setValue(String value) {
+            Widget valueWidget = valuePanel.getWidget();
+            if (valueWidget == priorityList)
+                priorityList.setSelectedIndex(getSelectedValueIndex(priorityList, value));
```

Comment:
should it be "setSelectedValue"? I see that the getValue() method above is returning the value

Reply:
`setSelectedValue` does not exist in GWT for dropdowns so I had to go through indexes to set the right value. This was satisfied by the `getSelectedValueIndex` method that I have added.